executor/load_data.go

```diff
@@ -261,14 +262,29 @@ func (e *LoadDataInfo) InsertData(prevData, curData []byte) ([]byte, bool, error
 			break
 		}
 	}
-	e.ctx.GetSessionVars().StmtCtx.AddRecordRows(uint64(len(rows)))
-	err := e.batchCheckAndInsert(rows, e.addRecordLD)
-	if err != nil {
-		return nil, reachLimit, err
+	if !txnBatchCheckIns {
+		err := e.CheckAndInsertOneBatch()
```

Comment:
after this PR, it seems only unit-test logic will step into here, how about this code snippet  and remove `txnBatchCheckIns` let it always be `true` and modify unit-test to test real check logic

Reply:
@lysu  @jackysp 
some local rought tests(see jira issue 3092 comments) show that batchget per packet may not always be fasttter,  should we make it configurable to use batch per transaction or packet?